client/src/js/views/DashboardDogsListDogTemplate.html

```diff
@@ -3,28 +3,58 @@
         <img class="dog__avatar" src="<%= avatar ? avatar : '/img/dog_stock.jpg' %>">
     </div>
     <div class="dog__info">
-        <p>
-            <a class="dog__name" href="/api/dogs/<%= display_id %>"><%= name %></a>
+        <p class="dog__name">
+            <a class="" href="/api/dogs/<%= display_id %>"><%= name %></a>
         </p>
         <% if (owner) { %>
         <p class="dog__owner">
             <%= _.startCase(owner) %>
         </p>
         <% } %>
-        <label>
-            <input class="hide" type="checkbox" name="checked_in" disabled<%= checked_in ? ' checked' : '' %>>
-            Checked <%= checked_in ? 'In' : 'Out' %>
-        </label>
-        <% if (checked_in) { %>
-        <label>
-            <input type="radio" name="status-<%= display_id %>"<%= checked_in ? ' checked' : '' %>> In Office
-        </label>
-        <label>
-            <input type="radio" name="status-<%= display_id %>"> On a walk
-        </label>
-        <label>
-            <input type="radio" name="status-<%= display_id %>"> Missing
-        </label>
+
+
+        <% if (checked_in) { %> 
+            <div class="button--sidebar dropdown">
+                <div class="dropdown__current-selection">
+                    <div class=" indicator current-selection__indicator indicator--<%= current_status_color %>"
+                    >
+                        <%= current_status_verbose %>
+                    </div> <!-- end .dropdown__status -->
+                    <div class="dropdown__arrow-container">
+                        <div class="dropdown__arrow">
+                             <object class="dropdown__arrow-svg" type="image/svg+xml" data="/img/arrow.svg"></object>
+                        </div>
+                    </div>
+                </div> <!-- end .dropdown__current-selection -->
+                <div class="dropdown__selections">
+                    
+                    <% if (checked_in) { %>  
```

Comment:
You can remove this `if` statement because there is the same parent if statement on [line 16](https://github.com/istrategylabs/paw/pull/38/files#diff-eb881147ba2e3c1830eb85b5b2b1dd8fR16).